react/src/components/ResourcePresetSettingModal.tsx

```diff
@@ -121,30 +141,60 @@ const ResourcePresetSettingModal: React.FC<ResourcePresetSettingModalProps> = ({
             },
           });
         } else {
-          commitModifyResourcePreset({
-            variables: {
-              name: values?.name,
-              props: props as ModifyResourcePresetInput,
-            },
-            onCompleted: (res, errors) => {
-              if (!res?.modify_resource_preset?.ok) {
-                message.error(res?.modify_resource_preset?.msg);
-                onRequestClose(false);
-              } else if (errors && errors?.length > 0) {
-                const errorMsgList = _.map(errors, (err) => err?.message);
-                _.forEach(errorMsgList, (err) => {
-                  message.error(err, 2.5);
-                });
-                onRequestClose(false);
-              } else {
-                message.success(t('resourcePreset.Updated'));
-                onRequestClose(true);
-              }
-            },
-            onError(err) {
-              message.error(err?.message);
-            },
-          });
+          if (
+            resourcePreset.id &&
+            baiClient.isManagerVersionCompatibleWith('25.4.0')
```

Comment:
We use the `updateSupportList` in esm file to show which features work for a particular version.

https://github.com/lablup/backend.ai-webui/blob/d4d8cce1ab501b2eb4fa03429dce20f4f37b4d5b/src/lib/backend.ai-client-esm.ts#L627

Reply:
And also, id field is  checked via @since in the gql schema as of version 25.4.0, so I think this part can be deleted.